src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -179,6 +182,34 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
               ),
             }}
           />
+          <NetworkSelector
+            value={formik.values.default_project_network}
+            setValue={(value) =>
+              void formik.setFieldValue("default_project_network", value)
+            }
+            hasNone
+            selectProps={{
+              label: "Default profile network",
+              disabled:
+                (features === "customised" &&
+                  (!formik.values.features_profiles ||
+                    formik.values.features_networks)) ||
```

Comment:
Can we isolate this into a const? Something like
```
const hasNoProfiles = features === "customised" && !formik.values.features_profiles;
const hasIsolatedNetworks = features === "customised" && formik.values.features_networks;
```
Then in here we can simplify the check to `hasNoProfiles || hasIsolatedNetworks || isEdit` which is much more readable.